packages/server/src/user/user.service.ts

```diff
@@ -13,17 +13,17 @@ export class UserService {
   /**
    * Register a new record of user in the database
    *
-   * @param data object that should contain `projectId: string`, `username?: string`, `email?: string` and `password: string` in plain text.
+   * @param newUser object that should contain `projectId: string`, `username?: string`, `email?: string` and `password: string` in plain text.
    * **NOTE:** `data` should contain either `username`, `email`, or both.
    * @returns User object, throws an `Error` when user already exist in the database
    */
-  async createUser(data: Prisma.UserCreateInput): Promise<User> {
+  async createUser(newUser: Prisma.UserUncheckedCreateInput): Promise<User> {
     // check if such user exist in the database
```

Comment:
Should we have a check to make sure the project ID is valid, or is it ok to assume the project ID is valid at this point?

Reply:
@gregfrasco said it will be handled by middleware. At this point, we assume the project ID exist and valid.

Reply:
Perfect, just wanted to make sure